berechenbarkeit-lib/src/vendors/regex.rs

```diff
@@ -141,13 +145,13 @@ impl RegexVendor {
         };
 
         Ok(InvoiceItem {
-            typ: InvoiceItemType::Expense,
+            typ: if ((net_price_single * amount) >= 0.0f64) { InvoiceItemType::Expense } else { InvoiceItemType::Credit },
             pos,
             article_number: groups.name("ARTNR").ok_or(InvoiceParseError::FieldMissingError("item.ARTNR".to_string()))?.as_str().to_string(),
             description: self.extract_description(groups)?,
-            net_price_single,
+            net_price_single: ((net_price_single / packaging_unit_amount) * 1000f64).round() / 1000f64,
```

Comment:
I'm unsure whether this is the right solution. Metro invoices have the result of this calculation as `Einzelpreis`. I _think_ that other invoices that have PUs would have that too